pages/settings/index.html

```diff
@@ -120,6 +120,52 @@ <h3 data-string="settingsPasswordAutoFillHeadline"></h3>
 
 	</div>
 
+	<div class="settings-container" id="proxy-settings-container">
+		<h3 data-string="settingsProxyHeading"></h3>
+
+		<div class="setting-section" id="proxy-type-container">
+			<select id="selected-proxy-type" style="min-width: 225px">
+				<option data-string="settingsNoProxy"></option>
+				<option data-string="settingsManualProxy"></option>
+				<option data-string="settingsAutomaticProxy"></option>
+			</select>
+		</div>
+
+		<div class="setting-section" id="proxy-grid">
+			<div class="setting-option">
+				<label for="http-proxy-host-input">HTTP</label>
+				<input id="http-proxy-host-input" type="text" name="httpHost" placeholder="proxy.example.com" />
+				<input type="number" id="http-proxy-port-input" name="httpPort" min="0" placeholder="8080" />
+			</div>
+			<div class="setting-option">
+				<label for="https-proxy-host-input">HTTPS</label>
+				<input id="https-proxy-host-input" type="text" name="httpsHost" placeholder="proxy.example.com" />
+				<input type="number" id="https-proxy-port-input" name="httpsPort" min="0" placeholder="8080" />
+			</div>
+			<div class="setting-option">
+				<label for="socks-proxy-host-input">SOCKS</label>
+				<input id="socks-proxy-host-input" type="text" name="socksHost" placeholder="proxy.example.com" />
+				<input type="number" id="socks-proxy-port-input" name="socksPort" min="0" placeholder="8080" />
+			</div>
+			<div class="setting-option">
+				<input id="socks4-input" type="radio" value="socks4" name="socksVersion" />
+				<label for="socks4-input">SOCKS v4</label>
+				<input id="socks5-input" type="radio" value="socks5" name="socksVersion" />
+				<label for="socks4-input">SOCKS v5</label>
+			</div>
+
+			<div class="setting-option">
+				<label for="proxy-bypass-rules" data-string="settingsProxyBypassRules"></label>
+				<input name="proxyBypassRules" id="proxy-bypass-rules" placeholder="localhost, 127.0.0.1/8, ::1" />
```

Comment:
I can see this placeholder potentially being confusing, because it implies that localhost is bypassed by default (is it?).

If it's not, the placeholder should probably something more generic, maybe like "hostname or IP address"?

Reply:
> ...because it implies that localhost is bypassed by default (is it?).

Yes, localhost is bypassed by default.